packages/manager/modules/iam/src/dashboard/applications/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "iam_applications_datagrid_column_application_id": "Application id",
+  "iam_applications_datagrid_column_application_name": "Application name",
+  "iam_applications_datagrid_column_application_description": "Description",
+  "iam_applications_datagrid_column_application_status": "Status",
+  "iam_applications_datagrid_column_application_status_active": "Active",
+  "iam_applications_datagrid_column_application_status_inactive": "Inactive",
+  "iam_applications_datagrid_column_application_key": "Application key",
+  "iam_applications_datagrid_action_delete_application": "Delete application",
+  "iam_applications_description": "View and manage your applications. For more information about applications, see the documentation."
+}
```

Comment:
English in fr translatation file ?

Reply:
Will be translated with the next pull request